### ISSUE TITLE
Zone texture (aka PBR cubemap) is not exposed in scene editor

### ISSUE THREAD

**Issue by darrylryan** [AtomicGameEngine/AtomicGameEngine#1174]
It looks like there was some effort to expose the zone texture to the editor but it isn't actually working:

https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Source/Atomic/Graphics/Zone.cpp#L88

**Comment by JoshEngebretson** [comment]
Let me know if you have some pointed questions on hooking this up, I have a reminder to make some comments here, though some guidance on what guidance may be needed would be helpful ;)

**Comment by dragonCASTjosh** [comment]
i have been trying to solve this for some time but i have no idea what is causing it as the issue only shows in this 1 case

**Comment by darrylryan** [comment]
@dragonCASTjosh Yep, I had a similar experience... I am wondering if its something to do with the GUI code  rather than the internals. I haven't had a look at that yet.

**Comment by dragonCASTjosh** [comment]
not sure as i remember there being other UI widgets created the same way of the same type but they work. it seems to be this unique case

**Comment by JoshEngebretson** [comment]
Ok, the cubemap is a TextureCube which is a ResourceRef:

https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Source/Atomic/Graphics/Zone.cpp#L88

This uses the ResourceRefAttributeEdit, which requires an importer:

https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Script/AtomicEditor/ui/frames/inspector/AttributeInfoEdit.ts#L794

The asset database doesn't know about an importer for TextureCube, so it is filtered in  the inspector:

https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Source/ToolCore/Assets/AssetDatabase.cpp#L649

Adding 

```C++
resourceTypeToImporterType_["TextureCube"] = "TextureImporter";
```

... and the Zone Texture shows up in inspector, however when clicking ... to select (and drag and drop operations) will work with any Texture2D and cube maps might not show up.

![screen shot 2016-11-22 at 4 17 50 pm](https://cloud.githubusercontent.com/assets/376203/20547184/4930d052-b0cf-11e6-9f51-ff7e7e0e8110.png)

Here's where the importers for various file types are setup (this should be dynamically populated, but hey):

https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Source/ToolCore/Assets/Asset.cpp#L272

Do you have enough info to go off now?

**Comment by dragonCASTjosh** [comment]
how would i go about making it accept XML as a cubemap file format

**Comment by JoshEngebretson** [comment]
I am thinking we need a .cubemap extension, much like .material is used for material xml so you, artists, and the editor can know what is inside, without having to open the file and investigate

**Comment by JoshEngebretson** [comment]
@darrylryan Did you plan to have this ready in time for Build 1, which is scheduled for November 30th?  If not, that is fine can push to Build 2

**Comment by JoshEngebretson** [comment]
Moved to Build 2
